components/panel/color.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import PanelBody from './body';
+
+function PanelColor( { colorValue, title, ...props } ) {
+	return (
+		<PanelBody
+			{ ...props }
+			title={ [
+				<span className="components-panel__color-title" key="title">{ title }</span>,
+				colorValue && <span className="components-panel__color-area" key="color" style={ { background: colorValue } } />,
+			] }
```

Comment:
I like the overall idea to show the selected color. However, this is just a visual indication. There's no text or label that can be read by software. It would be nice to find a way to add some meaningful ext, whether it's visually hidden text or an `aria-label` attribute to override the button text.